composer.json

```diff
@@ -11,9 +11,9 @@
     ],
     "require": {
         "php": "^7.4 || ^8.0",
-        "symfony/finder": "^4.2 || ^5.0 || ^6.0 || ^7.0",
-        "phpstan/phpstan": "^1.10.1",
+        "phpstan/phpstan": "1.10.x-dev",
```

Comment:
Will mark ready once https://github.com/phpstan/phpstan-src/commit/b87e5c4c7e33a91c61341f0335a221c32df603b2 is released